### ISSUE TITLE
NVDA reads the title of the individual links after expanding 3 times.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13776]
![image](https://user-images.githubusercontent.com/93735775/172650171-c289848e-7a96-441c-a681-b4dfd32d1227.png)

<img width="643" alt="image" src="https://user-images.githubusercontent.com/93735775/172650277-85f88630-2b53-47cd-9c45-9deae8c4a040.png">

### Steps to reproduce:
Step 1: Open the above URL in New Edge browser with valid credentials.
Step 2: Home Page gets displayed
Step 3: Validate whether the content of the ProTips for Gartner section is clearly read by NVDA.

### Actual behavior:
NVDA doesn’t read the title of links in ProTips for Gartner properly. It reads the title of the individual links after expanding 3 times.

### Expected behavior:
NVDA should read the titles of the links only once

#### NVDA installed/portable/running from source:
OS: Windows 11 Enterprise Version 21H2 (OS Build 22000..675)
Browser: Version 102.0.1245.33 (Official build) (64-bit)

#### NVDA version:
2021.3.5

#### Windows version:
OS: Windows 11 Enterprise Version 21H2 (OS Build 22000..675)

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

- Many developers, contributors and community members here are vision impaired. Screenshots are not accessible. Provide an alternative description, and capture any relevant information from images.
- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).

- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers
